amethyst_renderer/src/lib.rs

```diff
@@ -75,7 +75,7 @@ extern crate gfx_core;
 extern crate gfx_macros;
 extern crate num_cpus;
 extern crate rayon;
-extern crate serde;
+//extern crate serde;
```

Comment:
Why just comment this out? If we're going to remove it we should remove this line.